engine/api/application/application_importer.go

```diff
@@ -18,6 +18,14 @@ func Import(db gorp.SqlExecutor, store cache.Store, proj *sdk.Project, app *sdk.
 		return sdk.WrapError(erre, "application.Import> Unable to check if application exists")
 	}
 
+	if app.RepositoryStrategy.ConnectionType == "ssh" {
```

Comment:
Can you add this code directly to the exportentities -> sdk struct convert func.